examples/ConfigStoreDemo/Program.cs

```diff
@@ -24,9 +26,13 @@ public static IWebHost BuildWebHost(string[] args)
                     // 3. Set up the provider to listen for changes to the background color key-value in Azure App Configuration
 
                     var settings = config.AddJsonFile("appsettings.json").Build();
+
+                    IConfigurationSection endpointsSection = settings.GetSection("endpoints");
```

Comment:
>settings.GetSection("endpoints"); [](http://example.com/codeflow?start=61&length=33)

It might be helpful to add sample endpoints list in the appsettings.json file. Same for the ConsoleApplication.